solver/llbsolver/provenance/predicate.go

```diff
@@ -4,6 +4,7 @@ import (
 	"strings"
 
 	"github.com/containerd/containerd/platforms"
+	slsacommon "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/common"
 	slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
```

Comment:
Looks like the project itself uses `slsa02` as alias; let me update it here as well to stay consistent and prevent confusion.